src/rules/data-quality/schedule-contains-recurrence-data-rule.js

```diff
@@ -0,0 +1,88 @@
+const { RRule } = require('rrule');
+const Rule = require('../rule');
+const generateRRuleOptions = require('../../helpers/rrule-options');
+const getScheduleProperties = require('../../helpers/schedule-properties');
+const ValidationErrorType = require('../../errors/validation-error-type');
+const ValidationErrorCategory = require('../../errors/validation-error-category');
+const ValidationErrorSeverity = require('../../errors/validation-error-severity');
+
+module.exports = class ValidRecurrenceRule extends Rule {
+  constructor(options) {
+    super(options);
+    this.targetModels = ['Schedule'];
+    this.meta = {
+      name: 'ValidRecurrenceRule',
+      description:
+        'Validates that the Schedule contains the correct information to generate a valid iCal recurrence rule.',
+      tests: {
+        default: {
+          message:
+            'Schedule must contains the correct information to generate a valid iCal recurrence rule.',
+          sampleValues: {
+            startTime: '08:30',
+            endTime: '09:30',
+            startDate: '2021-03-19',
+            repeatFrequency: 'P1W',
+            count: 10,
+            scheduleTimezone: 'Europe/London',
+          },
+          category: ValidationErrorCategory.CONFORMANCE,
+          severity: ValidationErrorSeverity.FAILURE,
+          type: ValidationErrorType.MISSING_REQUIRED_FIELD,
+        },
+        dtStart: {
+          message:
+            'The recurrence rule must contain a startDate, startTime, and scheduledTimezone to generate the schedule.',
+          sampleValues: {
+            startTime: '08:30',
+            startDate: '2021-03-19',
+            scheduleTimezone: 'Europe/London',
```

Comment:
`sampleValues` are used to generate https://validator.openactive.io/about, so should match the placeholders in the message